src/content/code/language-support/go/client/go-graphql-client.md

```diff
@@ -0,0 +1,8 @@
+---
+name: graphql
```

Comment:
Should this be `go-graphql-client` or similar?

Reply:
yes @mikeesto , Made the change.